spec/support/vcr_cassettes/veteran_enrollment_system/form1095_b/get_form_success.yml

```diff
@@ -0,0 +1,98 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: <VETERAN_ENROLLMENT_SYSTEM_BASE_URI>/ves-ee-summary-svc/form1095b/1012667145V762142/2024
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Accept:
+      - application/json
+      Content-Type:
+      - application/json
+      User-Agent:
+      - Vets.gov Agent
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      Date:
+      - Thu, 01 May 2025 17:03:17 GMT
+      Content-Type:
+      - application/json
+      Transfer-Encoding:
+      - chunked
+      Connection:
+      - keep-alive
+      Server:
+      - envoy
+      X-Frame-Options:
+      - SAMEORIGIN
+      Content-Security-Policy:
+      - 'default-src https: data: ''unsafe-inline'' ''unsafe-eval'''
+      X-Oneagent-Js-Injection:
+      - 'true'
+      X-Envoy-Upstream-Service-Time:
+      - '43'
+      Ssl-Env:
+      - 'On'
+      X-Xss-Protection:
+      - 1; mode=block
+      X-Content-Type-Options:
+      - nosniff
+      Server-Timing:
+      - dtSInfo;desc="0", dtRpid;desc="-188950051"
+      Strict-Transport-Security:
+      - max-age=16000000; includeSubDomains; preload;
+    body:
+      encoding: UTF-8
+      string: |-
+        {
+          "data":{
+            "issuer":{
+              "issuerName":"US Department of Veterans Affairs",
+              "ein":"54-2002017",
+              "contactPhoneNumber":"877-222-8387",
+              "address":{
+                "street1":"PO Box 149975",
+                "city":"Austin",
+                "stateOrProvince":"TX",
+                "zipOrPostalCode":"78714-8975",
+                "country":"USA"
+              }
+            },
+            "responsibleIndividual":{
+              "name":{
+                "firstName":"VGSSFIFTYSIX",
+                "lastName":"TESTFIFTYSIX"
```

Comment:
Where is this test user from? I don't see this user's name or SSN on https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-users.csv so I just want to be extra sure it's a test user.

Reply:
Good question. When Josh Faulkner gave me API keys for the enrollment system, he also gave me an example curl request using the api key. This is the result of that curl request. It's definitely a test user on the enrollment system. Maybe it doesn't exist on the vets-api.

However, your question made me wonder about the rest of this test data. The address appears to be real, which makes me wonder if this data was copied from real users and anonymized. Should I change the data to fake addresses and a new made up SSN?